mne/commands/tests/test_commands.py

```diff
@@ -177,4 +178,34 @@ def test_surf2bem():
     check_usage(mne_surf2bem)
 
 
+@slow_test
+@requires_freesurfer
+@sample.requires_sample_data
+def test_watershed_bem():
+    """Test mne watershed bem"""
+    check_usage(mne_watershed_bem)
+    orig_subject_dir = os.getenv('SUBJECTS_DIR', None)
+    orig_subject = os.getenv('SUBJECT', None)
```

Comment:
Is it actually necessary at all to get and set `os.environ[...]` here in test_commands.py?
In any case I pass SUBJECTS_DIR and SUBJECT as arguments and the checks to `os.environ` are done inside the function `make_watershed_bem` (where, following your comments below, I shouldn't modify at all `os.environ`).